industrial_ci/source_tests.sh

```diff
@@ -246,7 +228,7 @@ if [ "$NOT_TEST_BUILD" != "true" ]; then
     fi
 
     if [ "$BUILDER" == catkin ]; then
-        source devel/setup.bash ; rospack profile # force to update ROS_PACKAGE_PATH for rostest
```

Comment:
Looks like this was originally added at https://github.com/jsk-ros-pkg/jsk_travis/pull/164
Trying to figure out why we no longer need this, but I'm not sure yet.